java-weblogic/README.md

```diff
@@ -0,0 +1,50 @@
+# Quickstart for Java + Rookout and Oracle WebLogic
+
+Before following this guide we recommend reading the basic [Java + Rookout] guide
+
+* [Running locally](#running-locally)
+* [Using docker compose](#using-docker-compose)
+* [Rookout Integration explained](#rookout-integration-explained)
+## Running locally
+1. Run the Rookout agent:
+    ``` bash
+    $ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
+    ```
+1. Add the Rookout javaagent as described in [Rookout Integration Explained](#rookout-integration-explained)
+1. Build the sample helloworld WebLogic application:
+    ``` bash
+    $ cd weblogic-helloworld
+    $ ant
+    ```
+    The built web application is now situated in `build/weblogic-helloworld.war`
+1. Start your WebLogic domain `sh WEBLOGIC_HOME/user_projects/domains/DOMAIN_NAME/bin/startWebLogic.sh`
+1. Deploy to WebLogic using the console (default url [http://localhost:7001/console](http://localhost:7001/console))
+1. Go to [app.rookout.com](https://app.rookout.com/) and start debugging !
+
+## Rookout Integration explained
+
+This example is based of the Java javalin "Hello-World" example available [here].
+
+We have added Rookout to the original project by:
+1. Adding source files to the project .war in the ant task:
+    ```xml
+       <fileset dir="." includes="${source-directory}/**" />
+    ```
+1. Download javaagent available on [maven central]:
+    ```bash
+        wget "http://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.rookout&a=rook&v=LATEST"  -O rook.jar
+    ```
+1. Specified the javaagent in Weblogic startup script:
```

Comment:
@itielshwartz there's not really an easier automated way to do this that I found out about. We can merge this solution and try to find a better way in the future

Reply:
I mean it's possible to do export JAVA_OPTIONS=$JAVA_OPTIONS -javaagent=ROOKLOCATION/rook.jar but is it better?